src/Suave/Suave.fsproj

```diff
@@ -196,4 +196,89 @@
       </ItemGroup>
     </When>
   </Choose>
+  <Choose>
```

Comment:
Sorry, I'm new to all of this. Does making it a stand-alone lib mean that I will put the json parsing code into a separate dll?